object.c

```diff
@@ -3728,16 +3759,30 @@ rb_Hash(VALUE val)
 
 /*
  *  call-seq:
- *     Hash(arg)    -> hash
+ *    Hash(object) -> object or new_hash
+ *
+ *  Returns a hash converted from +object+.
+ *
+ *  If +object+ is:
+ *
+ *  - A hash, returns +object+.
+ *  - An empty array or +nil+, returns an empty hash.
+ *
+ *  Otherwise, if +object+ responds to:
+ *
+ *  - +to_hash+, returns <tt>object.to_hash</tt>.
+ *  - +to_h+, returns <tt>object.to_h</tt>.
+ *
+ *  Otherwise, returns TypeError.
+ *
+ *  Examples:
  *
- *  Converts <i>arg</i> to a Hash by calling
- *  <i>arg</i><code>.to_hash</code>. Returns an empty Hash when
- *  <i>arg</i> is <tt>nil</tt> or <tt>[]</tt>.
+ *    Hash({foo: 0, bar: 1}) # => {:foo=>0, :bar=>1}
+ *    Hash(nil)              # => {}
+ *    Hash([])               # => {}
+ *    Hash([0, 1, 1])        # Raises TypeError; array not empty.
```

Comment:
Please remove the examples raising TypeError.

Reply:
Fixed.